core/server/api/v2/utils/serializers/input/tags.js

```diff
@@ -1,7 +1,30 @@
 const debug = require('ghost-ignition').debug('api:v2:utils:serializers:input:tags');
 const url = require('./utils/url');
+const utils = require('../../index');
+
+function setDefaultOrder(frame) {
+    if (!frame.options.order) {
+        frame.options.order = 'name asc';
+    }
+}
 
 module.exports = {
+    browse(apiConfig, frame) {
+        debug('browse');
+
+        if (utils.isContentAPI(frame)) {
+            setDefaultOrder(frame);
+        }
+    },
+
+    read(apiConfig, frame) {
+        debug('read');
+
+        if (utils.isContentAPI(frame)) {
```

Comment:
> this. browse(...arguments);

Reply:
Hm.. now that I searched for it, see it's been used in other places as well. Will do the change change, but the debug logs will become a bit confusing because of such calls I think. You'd see:
```
'read'
'browse'
```
One after another :)